contributing.md

```diff
@@ -113,9 +113,7 @@ You might encounter a `Unsupported platform` error when you run `npm install`. T
 
 ```shell
 # Delete node_modules in sub-directories
-find ./ -type d -name "node_modules" -print0 | while read -d $'\0' dir; do
-  rm -rf "$dir"
-done
+find ./ -name "node_modules" -type d -exec rm -rf {} +
```

Comment:
@MNR85 Can you share more details of how this update is helpful?

The current code is more readable in my opinion. We find all directories having name as `node_modules`, print it to console, then do a while loop where we are deleting each directory

With the new change, its not as readable. I don't know what ` {} +` means